src/App.tsx

```diff
@@ -1,73 +1,70 @@
-import React from 'react';
+import React, { useMemo, useState } from 'react';
 import './App.scss';
+import classNames from 'classnames';
+
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
+
+import { SearchInput } from './components/searchInput/searchInput';
+import Content from './components/content/content';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [delay] = useState(300); // for change add setDelay
```

Comment:
The `useState` hook for `delay` is missing the setter function. According to the task checklist, you should use both the state and its setter if you plan to change the state value. Consider adding `setDelay` if you intend to modify `delay` later.